WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt

```diff
@@ -0,0 +1,27 @@
+this_is_not_sparta = { # Spartan cultured character is a coward, AKA unthinkable in Spartan culture
+	icon = 90
+
+	general_opinion = -30
+	monthly_character_prestige = -1
+
+	martial = -20 # Prevents a Spartan AI ruler from choosing characters with the Craven Trait as Commanders
+
+#	revoke_reason = yes # Cowardly Spartans can be relieved of their duties by their liege
+#	banishment_reason = yes # Cowardly Spartans were expelled from Sparta if they couldn't man up
+}
+
+inferior_spartan_child = { # A deformed or otherwise unworthy child was left by Spartan society to die, or roll back down the hill and be embraced as a Spartan
+	icon = 42
+
+	martial = -5 # Negates Spartan cultural bonus
+	pagan_opinion = -30
```

Comment:
Okay. Make sure you add a "SPARTAN_OPINION" for the Spartan culture as well, plus an "ANTINOMIANISM_OPINION". In a future PR, I'm going to add opinions so that Antinomians are not effected by the maluses to adultery and the like, etc. .